src/adapter/src/coord.rs

```diff
@@ -1729,15 +1710,15 @@ impl Coordinator {
         //
         // NOTE: If we ever allow custom index compaction windows, we'll need to apply those here
         // as well.
-        let lag = if is_retained_metrics_index {
-            let retention = self.catalog().state().system_config().metrics_retention();
-            Timestamp::new(u64::try_from(retention.as_millis()).unwrap_or_else(|_| {
-                tracing::error!("absurd metrics retention duration: {retention:?}");
-                u64::MAX
-            }))
+        let retention = if is_retained_metrics_index {
```

Comment:
The comment above this mentions a default 1-second compaction window, which I think is no longer accurate.